Libplanet/Action/PolymorphicAction.cs

```diff
@@ -243,7 +243,7 @@ public string GetInnerActionTypeName()
 
         public void LoadPlainValue(Dictionary plainValue)
         {
-            var typeStr = (Text)plainValue["type_id"];
+            var typeStr = plainValue["type_id"];
```

Comment:
It doesn't have to be updated in this pull request, the variable name can be renamed into another like `typeId` or.